src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -113,11 +113,15 @@
             new String[] {String.valueOf(1)}, null, getClass(), null);
 
     private static String getPath(String filename) {
-        return "src/test/resources/com/puppycrawl/tools/checkstyle/" + filename;
+        return "src/test/resources/com/puppycrawl/tools/checkstyle/main/" + filename;
+    }
+
+    private static String getNonPackagePath(String filePath) {
+        return "src/test/resources/com/puppycrawl/tools/checkstyle/" + filePath;
```

Comment:
So did I. For `checks/metrics` see https://github.com/checkstyle/checkstyle/pull/4697#discussion_r126657978.
We should use MainTest's input folder, not someone elses.